data-raw/run_and_compare_templates.R

```diff
@@ -0,0 +1,67 @@
+library(purrr)
+library(cli)
+library(stringr)
+library(devtools)
+devtools::install_github("pharmaverse/pharmaversesdtm", ref = "main")
+devtools::install_github("pharmaverse/pharmaverseadam", ref = "main")
+library(pharmaverseadam)
+# nolint start
+
+# Gather templates ----
+template_dir <- file.path("inst/templates")
+templates <- list.files(template_dir, pattern = "ad_")
+
+# From templates generate vector of adam_names ----
+adam_names <- vapply(
+  templates, function(x) gsub("ad_|\\.R", "", x),
+  USE.NAMES = FALSE, character(length = 1)
+)
+
+# Ignore ADLBHY
+adam_names <- adam_names[adam_names != "adlbhy"]
+
+# Run templates and compare ----
+diffs <- purrr::map(adam_names, function(adam) {
+  cli_inform("Running {adam} template and comparing with pharmaverseadam version...")
+
+  ## Get {pharmaverseadam} dataset ----
+  data_env <- new.env()
```

Comment:
Working with .rda files:   This is fine.     Since have found a few tricks:     get(load(...) reduces need for explicit new environment.   Code as it is clear.